src/scene/shader-lib/chunks/standard/frag/frontend.js

```diff
@@ -0,0 +1,37 @@
+export default /* glsl */`
+struct Frontend {
```

Comment:
I also wonder if we should have a better name than Frontend, considering this is going to be a public API.
Unity calls it SurfaceOutput for example.
Maybe SurfaceProperties? Or MaterialProperties?